packages/eslint-config-airbnb-base/rules/best-practices.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     'consistent-return': 'error',
 
     // specify curly brace conventions for all control statements
-    curly: ['error', 'multi-line'],
+    curly: ['error', 'multi-line'], // multiline
```

Comment:
this is the only spot where I didn't make the actual change because not sure if it would break anything 😨

Reply:
indeed, it would :-)